pyanaconda/rescue.py

```diff
@@ -37,11 +37,13 @@
 import os
```

Comment:
The commit comment ``... would be to make the UI "will" more sane ..`` is confusing. I guess the `will` should not be there.